core/variant/variant_utility.cpp

```diff
@@ -484,6 +484,14 @@ struct VariantUtilityFunctions {
 		return str;
 	}
 
+	static inline String error_string(int error) {
+		if (error < 0 || error >= ERR_MAX) {
+			return String("Invalid error code");
```

Comment:
Make it clear that "Invalid error code" is not an error code message, but an internal error:

```suggestion
			return String("(invalid error code)");
```